test/cypress/integration/web-payments/apple-pay.test.cy.js

```diff
@@ -89,7 +89,21 @@ describe('Apple Pay payment flow', () => {
     adminUsersGetService()
   ]
 
-  const checkCardDetailsStubsWithApplePayorGooglePay = (applePayEnabled, googlePayEnabled, emailCollectionMode = 'MANDATORY', agreement, moto) => {
+  const createPaymentChargeStubsWelsh = [
+    connectorFindChargeByToken({ tokenId }),
+    connectorMarkTokenAsUsed(tokenId),
+    connectorGetChargeDetails({
+      chargeId,
+      status: 'CREATED',
+      state: { finished: false, status: 'created' },
+      paymentProvider: 'worldpay',
+      language: 'cy'
+    }),
+    connectorUpdateChargeStatus(chargeId),
+    adminUsersGetService()
+  ]
+
+  const checkCardDetailsStubsWithApplePayorGooglePay = (applePayEnabled, googlePayEnabled, emailCollectionMode = 'MANDATORY', agreement, moto, language = 'en') => {
```

Comment:
`checkCardDetailsStubsWithApplePayOrGooglePay` (I know it was like this already but may as well fix it while we’re here).

Reply:
Do you just mean fixing `or` -> `Or`?

Reply:
Yes.

Reply:
Fixed